tsinfer/formats.py

```diff
@@ -2333,6 +2355,26 @@ def sequence_length(self):
     def num_sites(self):
         return self._num_sites
 
+    @functools.cached_property
+    def individuals_select(self):
+        if self._sgkit_samples_mask_name is None:
```

Comment:
Same comment as sites_select - can we just compute and store this at init time?

Reply:
Fixed in https://github.com/tskit-dev/tsinfer/pull/896/commits/56df80090c8fcb36bc18ca192f97df063d4f51b2